src/components/auth/Profile.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { NavLink, Outlet } from "react-router-dom";
+import "../stylesheets/profile.css";
+
+const navLinkStyle = ({ isActive }) => {
+  return {
+    borderBottom: isActive ? "#392F5A solid 3px" : "",
```

Comment:
you can create class for borderbottom property instead of inline.

Reply:
as you suggested I've created class for styling active tabs
thank you !